CHANGELOG.md

```diff
@@ -1,4 +1,4 @@
-# Release 0.17.1
+# Release 0.17.2
```

Comment:
I don't care much, but do we want to leave a note about the 0.17.1 release being non-existent so folks aren't left wondering?

Reply:
I like the mystique of it.

Reply:
they can read between the lines